src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
```

Comment:
```suggestion
const body = document.body;
```
До `body` можна простіше достукатись)

Reply:
точно, щось дуже задумався)